MozillaFoundation/foundation.mozilla.org
# NETWORK_SITE_URL env var removal from files

# Description

This PR removes NETWORK_SITE_URL env var from codebase and is replaced for origin property of the Location interface `window.location.origin`. 

# How to test
- Due to there hasn't been a separate Network API for years (as the issue specifies), the way to test these changes is only making sure at the time we merge into staging the deletion of this var is not casuing any conflicts, and then remove also from prod.
- Another way to test is is reviewing jobs from the from the continous-integration built properly. As you can see on previous commits under this PR some of the jobs failed due to NETWORK_SITE_URL removal from the github workflows, but it was already fixed using `window.location.origin`. 

Link to sample test page: https://foundation-s-tp1-250-re-ubeqtp.herokuapp.com/en/
Related PRs/issues: https://github.com/MozillaFoundation/foundation.mozilla.org/issues/7613

# Checklist



**Tests**

**Changes in Models:**

**Documentation:**

**Merge Method**
**💡❗Remember to use squash merge when merging non-feature branches into `main`**

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-1065)
